commands/tx-status.js

```diff
@@ -0,0 +1,37 @@
+const exitOnError = require('../utils/exit-on-error');
+const connect = require('../utils/connect');
+const inspectResponse = require('../utils/inspect-response');
+const bs58 = require('bs58');
+
+module.exports = {
+    command: 'tx-status <hash>',
+    desc: 'lookup transaction status by hash',
+    builder: (yargs) => yargs
+        .option('hash', {
+            desc: 'base58-encoded hash',
+            type: 'string',
+            required: true
+        }),
+    handler: exitOnError(async (argv) => {
+        const near = await connect(argv);
+
+        const hashParts = argv.hash.split(':');
+        let hash, accountId;
+        if (hashParts.length == 2) {
+            [accountId, hash] = hashParts;
+        } else if (hashParts.length == 1) {
+            [hash] = hashParts;
+        } else {
+            throw new Error('Unexpected transaction hash format');
+        }
+        accountId = accountId || argv.accountId || argv.masterAccount;
+
+        if (!accountId) {
```

Comment:
in this command, accountId is actually not required and user will only follow the command we give not given the account id by default? Maybe change instructions in the readme for this since if not accountId, there will throw an error.

Reply:
the intent is to change other tools to return transaction hash as `accountId:hash`, so we can just have one string that can be copy-pasted

see https://github.com/nearprotocol/near-shell/issues/187